Gopkg.toml

```diff
@@ -142,7 +142,7 @@ required = [
   name = "github.com/nats-io/nats-streaming-server"
   version = "0.11.0"
 
-# The latest release as of March 21, 2019.
 [[constraint]]
   name = "github.com/cloudevents/sdk-go"
-  version = "=0.4.3"
+  # HEAD as of 2019-03-21
+  revision = "b88e53efaaadf77c676a5b53f64c2dc8814e1c6a"
```

Comment:
perhaps we work on getting a release? /cc @n3wscott

Reply:
Done